documents/Libraries/stdlib/stdlib_sx-glsl_impl.mtlx

```diff
@@ -15,24 +15,52 @@
   <!-- <image> -->
   <implementation name="IM_image_float_sx_glsl" nodedef="ND_image_float" file="stdlib/sx-glsl/mx_image_float.glsl" function="mx_image_float" language="sx-glsl">
     <parameter name="default" type="float" implname="default_value"/>
+    <parameter name="uaddressmode" type="string" implname="uaddressmode" impltype="integer" enumvalues="0,1,2"/>
```

Comment:
Map uaddressmode string to uaddressmode integer with matching enumerants to those found on the nodedef.

Reply:
Don't really care what the number is so use 0 ascending as a default enum would. It would be much easier if this was not required and enum types kept an enum / index number as the value.